spec/services/hyrax/collection_types/permissions_service_spec.rb

```diff
@@ -1,298 +1,295 @@
 # frozen_string_literal: true
 RSpec.describe Hyrax::CollectionTypes::PermissionsService do
-  let(:user_cg) { create(:user, groups: 'create_group') }
-  let(:user_mg) { create(:user, groups: 'manage_group') }
-  let(:user_cu) { create(:user) }
-  let(:user_mu) { create(:user) }
-  let(:user_other) { create(:user) }
-  let(:user_admin) { create(:user, groups: 'admin') }
-
-  let!(:user_collection_type) { create(:user_collection_type) }
-  let!(:admin_set_collection_type) { create(:admin_set_collection_type) }
-  let!(:collection_type) { create(:collection_type, creator_user: user_cu, creator_group: 'create_group', manager_user: user_mu, manager_group: 'manage_group') }
+  let(:user_cg) { FactoryBot.create(:user, groups: 'create_group') }
+  let(:user_mg) { FactoryBot.create(:user, groups: 'manage_group') }
+  let(:user_cu) { FactoryBot.create(:user) }
+  let(:user_mu) { FactoryBot.create(:user) }
+  let!(:user_collection_type) { FactoryBot.create(:user_collection_type) }
+  let!(:admin_set_collection_type) { FactoryBot.create(:admin_set_collection_type) }
+
+  let!(:collection_type) do
```

Comment:
Definitely like this change to improve the ability to scan this object's parameters.